lib/_http_outgoing.js

```diff
@@ -628,9 +628,9 @@ function matchHeader(self, state, field, value) {
   }
 }
 
-const validateHeaderName = hideStackFrames((name) => {
+const validateHeaderName = hideStackFrames((name, label = 'Header name') => {
```

Comment:
This does not work the way you want, as it will be `undefined` if the second parameter is explicitly passed as `undefined`. Move it to a `||` conditional.

Reply:
Applied your suggestion. Thank you for your help.

Reply:
FWIW it's not correct, the default value is used no matter if `undefined` is passed explicitly or not.

```js
((param = 'default value') => console.log(param))(); // 'default value'
((param = 'default value') => console.log(param))(undefined); // 'default value'
((param = 'default value') => console.log(param))(null); // null
```